ehr/src/org/labkey/ehr/EHRServiceImpl.java

```diff
@@ -1050,4 +1051,14 @@ public void appendSNOMEDCols(AbstractTableInfo ti, String displayColumnName, Str
             ti.addColumn(simpleRawCol);
         }
     }
+
+    public void registerLabWorkOverrides(Module module, String fromType, LabworkType toType)
+    {
+        _labWorkOverrides.put(fromType, Pair.of(module, toType));
```

Comment:
While we don't have conflicting overrides yet, this doesn't feel right in terms of how it treats the Module argument. Whichever module registers last will win. And the usage in LabworkManager totally ignores the Module.

I think _labworkOverrides should be a Map<String, Map<Module, LabworkType>>. And then when we look at the overrides, we should only apply those for modules that are active in the container.